docs/backend-system/building-plugins-and-modules/01-index.md

```diff
@@ -159,7 +159,7 @@ The same applies for modules that perform their own migrations and interact with
 the database. They will run on the same logical database instance as the target
 plugin, so care must be taken to choose table names that do not risk colliding
 with those of the plugin. A recommended naming pattern is `<package
-name>__<table name>`, for example the `@backstage/backend-tasks` package creates
+name>__<table name>`, for example the `Scheduler Service` package creates
```

Comment:
Is this something that we need to update to plugin api?

Reply:
Not in this particular text though since it discusses things that are modules side-by-side with a plugin, and what they do to not get colliding table names with the host plugin.

Yeah not sure what to best do here with the wording, since this one is a bit odd. The only other example I know is the incremental ingestion catalog module, but that one didn't follow the recommendations of using a prefix so that one is even worse :P OK keep this example but maybe say something like "for example the scheduler core service creates tables named `backstage_backend_tasks__<table>`, because it used to be the case that the service lived in a package named `@backstage/backend-tasks`. Things have since moved around a bit, but the effects of the rule are still visible."